pleasew

```diff
@@ -8,7 +8,7 @@ RESET="\x1B[0m"
 
 DEFAULT_URL_BASE="https://get.please.build"
 # We might already have it downloaded...
-LOCATION=$(grep -i "^location" .plzconfig 2>/dev/null | cut -d '=' -f 2 | tr -d ' ')
+LOCATION=$(grep -i "^location" ".plzconfig" "$HOME/.config/please/plzconfig" 2>/dev/null | head -n 1 | cut -d '=' -f 2 | tr -d ' ')
```

Comment:
Oh nice idea here using head to find the result. 

The Please config locations are, in order of increasing precedence: 

``` bash
/etc/please/plzconfig
~/.config/please/plzconfig
.plzconfig
.plzconfig.<profile> # this one might be a little tricky unless we can extract --profile or -p from $PLZ_ARGS or $@
.plzconfig_$OS_$ARCH # these vars are defined in this script
.plzconfig.local
```

I wonder if we can create a little bash function with your `grep | head` trick to look up arbitrary values in these files. We can use this to look up the version (L34/36), as well as the location here. 

What do you think? Doesn't look too tricky.

Reply:
Not too bad - i'm making the assumption that PROFILE never has a space in it with the latest changes - is that fair?

Reply:
Yeah, profiles can't have spaces in them. :)